### PR TITLE
docker-machine-driver-vultr 1.2.2 (new formula)

### PR DESCRIPTION
**Pull request by f-a-a**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

**Comment by ilovezfs** [comment]
@BrewTestBot test this please

File: Formula/docker-machine-driver-vultr.rb [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  depends_on "godep" => :build
+  depends_on "docker-machine" => :recommended
+
+  def install
+    ENV["GOPATH"] = buildpath
+    (buildpath/"src/github.com/janeczku/docker-machine-vultr").install buildpath.children
+
+    cd "src/github.com/janeczku/docker-machine-vultr" do
+      system "make"
+      bin.install "build/docker-machine-driver-vultr-v1.2.2" => "docker-machine-driver-vultr"
```
Comment (dunn) [inline Formula/docker-machine-driver-vultr.rb]: Here it's probably a good idea to replace `v1.2.2` with `#{version}` so anyone updating the formula in future doesn't get tripped up.

File: Formula/docker-machine-driver-vultr.rb [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    end
+  end
+
+  test do
+    assert_match "--vultr-api-endpoint",
+      shell_output("#{Formula["docker-machine"].bin}/docker-machine create --driver vultr -h")
```
Comment (dunn) [inline Formula/docker-machine-driver-vultr.rb]: I don't suppose it's possible to have a more thorough test?

Comment (f-a-a) [inline Formula/docker-machine-driver-vultr.rb]: I based the test here according to this https://github.com/Homebrew/homebrew-core/blob/master/Formula/docker-machine-driver-xhyve.rb#L72.

**Comment by dunn** [comment]
Merged as c90943524a6eb34cc1753655567dd505f5ce558e, thanks!

**Pull request closed without merging**